GetARock/GetARock/Screens/BandCreation/PracticeSongAddFlow/AddPracticeSongViewController.swift

```diff
@@ -0,0 +1,248 @@
+//
+//  AddPracticeSongViewController.swift
+//  GetARock
+//
+//  Created by 장지수 on 2023/03/09.
+//
+
+import UIKit
+
+final class AddPracticeSongViewController: BaseViewController {
+    
+    // MARK: - Property
+    
+    private var keyBoardHeight: CGFloat = 280
+    
+    // 정보 전달 클로저
+    var completion: (_ songs: [PracticeSongCardView]) -> Void = { songs in }
+    
+    private var numberOfSong: Int = 1 {
+        didSet {
+            addPracticeSongButton.configuration?.title = "합주곡 추가\(numberOfSong)/3"
+            addPracticeSongButton.configuration?.attributedTitle?.font = UIFont.setFont(.contentBold)
+            if numberOfSong == 3 {
+                addPracticeSongButton.isHidden = true
+            } else {
+                addPracticeSongButton.isHidden = false
+            }
```

Comment:
p4;
삼항연산자로 나타낼 수 있을 것 같아요!
```suggestion
                addPracticeSongButton.isHidden = numberOfSong == 3 ? true : false
```

Reply:
감사합니다 반영완료!! https://github.com/extreme-rock/GetARock-iOS/pull/158/commits/f8c0504f5e97380a7901985981280938a6d4ace5